.github/workflows/deployDev.yml

```diff
@@ -18,28 +31,47 @@ jobs:
       REACT_APP_VERSION: 'latest'
       REACT_APP_GOOGLE_ANALYTICS_ID: ${{ secrets.GOOGLE_ANALYTICS_ID }}
       REACT_APP_GRAASP_DOMAIN: ${{ secrets.DEV_GRAASP_DOMAIN }}
+    
+    # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-      - name: Checkout
+      - name: Check out code
```

Comment:
I think JC likes to have only lowercase

Reply:
The names of the steps are normally written with uppercase